README.md

```diff
@@ -2,7 +2,22 @@
 
 This repository is a monorepo for smart-contract development in AssemblyScript for Massa blockchain.
 
-## Build
+## Packages:
+
+### @massalabs/sc-project-initializer
+
+This package makes your first steps into massa smart-contracts development easy:
```

Comment:
Sc-initializer (smart-contract initializer), is a package that should be used as a first step when starting to develop smart contracts on Massa blockchain. It consists of the files necessary for setting up the environment for developing a smart-contract project.